colossalai/pipeline/policy/bert.py

```diff
@@ -243,8 +251,9 @@ def custom_forward(*inputs):
 class BertModelPolicy(Policy):
 
     def __init__(self, stage_manager: PipelineStageManager, num_layers: int, num_stages: int):
+        super().__init__(stage_manager=stage_manager)
         self.stage_manager = stage_manager
-        self.layers_per_stage = self.distribute_layers(num_layers, num_stages)
+        self.layers_per_stage = super().distribute_layers(num_layers, num_stages)
```

Comment:
You can call `self.distribute_layers()` directly.

Reply:
done